### DISCUSSION TITLE
I've made a translation: no_no (Norsk Boknål)

### DISCUSSION THREAD

**Discussion by Bloodaxe95** [vectorwing/FarmersDelight#396] [General]
Hi! I've translated the newest Farmers Delight release to my native language, Norsk Bokmål. I've never tried to get a translation added to a mod before, how does one usually go about to get the translation added to the official release? Do I have to make a pull request or something?

**Comment by vectorwing** [comment]
Heyo! Thanks for translating FD! ^^

You can submit the file through a pull request, if you're familiar with forking repositories; it helps me proof-check the translation for potential problems.
If you're not sure how, you can also just send me the .json file directly and I'll integrate it. 👍

**Reply by Bloodaxe95** [reply]
Gonna try the pull request thingy then! Gotta learn how to do it at some point ^^

**Reply by Bloodaxe95** [reply]
Ok, so I did a fork + pull request, was this the right way of doing things? The translation was only tested in 1.18.1, so that's where I committed it.

**Reply by vectorwing** [reply]
Yes, that's it! Thanks for the PR, I'll review it later today after work.
As for other versions, 1.16 will be abandoned soon-ish, so I can always backport translations myself.
